main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	fmt.Print("> ")
+	scanner := bufio.NewScanner(os.Stdin)
+	for scanner.Scan() {
+		input := scanner.Text()
+		element := strings.Split(input, " ")
+		if len(element) != 3 {
+			fmt.Printf("Missing arguments or way too many arguments! %s\n", element)
+			os.Exit(1)
+		}
+		execute(element[0], element[1], element[2])
+		fmt.Printf("\n> ")
+	}
+}
+
+func execute(first, operand, second string) {
+	var result float64
+
+	firstNumber, err := strconv.Atoi(first)
+	if err != nil {
+		fmt.Println("First input must be an integer!")
+		os.Exit(1)
+	}
+
+	secondNumber, err := strconv.Atoi(second)
+	if err != nil {
+		fmt.Println("Second input must be an integer!")
+		os.Exit(1)
+	}
+
+	switch operand {
+	case "+":
+		result = float64(firstNumber) + float64(secondNumber)
+	case "-":
+		result = float64(firstNumber) - float64(secondNumber)
+	case "*":
+		result = float64(firstNumber) * float64(secondNumber)
+	case "/":
+		result = float64(firstNumber) / float64(secondNumber)
```

Comment:
what if `secondNumber` is zero.

Reply:
I see. I will fix this in the next commit ^^